Setup/Prepare-Release.ps1

```diff
@@ -147,10 +147,10 @@ try {
     Write-Host ----------------------------------------------------------------------
     Generate-Changelog -milestones $milestoneNumbers
 
-    #Write-Host ----------------------------------------------------------------------
-    #Write-Host Download PluginManager
-    #Write-Host ----------------------------------------------------------------------
-    #.\Download-PluginManager.ps1 -ExtractRootPath '..\Plugins\GitExtensions.PluginManager'
+    Write-Host ----------------------------------------------------------------------
+    Write-Host Download PluginManager
+    Write-Host ----------------------------------------------------------------------
+    .\Download-PluginManager.ps1 -ExtractRootPath '..\Plugins\GitExtensions.PluginManager'
```

Comment:
https://github.com/gitextensions/gitextensions/blob/775d297e583625ea1098eed6e7b58f0f78d2e403/Setup/Download-PluginManager.ps1#L10
`Download-PluginManager.ps1` still is configured to download from `maraf/GitExtensions.PluginManager`. This is no longer in line with the new approach for the PluginManager and we would skip a few important steps.

I think we should move in order:
- first set up `gitextensions/GitExtensions.PluginManager`
- review the code (at least I for myself never really looked into it)
- build a release from the reviewed code under `gitextensions/GitExtensions.PluginManager`
- include it in the installer

Reply:
We'll need to have it signed too